package.json

```diff
@@ -14,17 +14,24 @@
     "access": "public"
   },
   "dependencies": {
+    "@kartotherian/err": "^0.0.3",
+    "@kartotherian/input-validator": "^0.0.6",
+    "@kartotherian/module-loader": "^0.0.5",
     "bluebird": "^3.5.0",
     "js-yaml": "^3.8.2",
-    "underscore": "^1.8.3",
+    "npm": "^5.7.1",
```

Comment:
why do you need npm dependency?

Reply:
I don't remember adding it... could it have been added when I ran npm install?! yeesh. I'll take it out.